samples/app/MeshAssimp.cpp

```diff
@@ -647,21 +686,128 @@ bool MeshAssimp::setFromFile(const Path& file, std::vector<uint32_t>& outIndices
     size_t depth = 0;
     size_t matCount = 0;
 
-    const std::function<void(aiNode const* node, int parentIndex)> processNode =
-            [&](aiNode const* node, int parentIndex) {
+    if (scene) {
+        aiNode const* node = scene->mRootNode;
 
-        mat4f const& current = transpose(*reinterpret_cast<mat4f const*>(&node->mTransformation));
+        size_t totalVertexCount = 0;
+        size_t totalIndexCount = 0;
 
-        size_t totalIndices = 0;
-        outParents.push_back(parentIndex);
-        outMeshes.push_back(Mesh{});
-        outMeshes.back().offset = outIndices.size();
-        outMeshes.back().transform = current;
+        countVertices(node, totalVertexCount, totalIndexCount);
 
-        mat4f parentTransform = parentIndex >= 0 ? outMeshes[parentIndex].accTransform : mat4f();
-        outMeshes.back().accTransform = parentTransform * current;
+        asset.positions.reserve(asset.positions.size() + totalVertexCount);
+        asset.tangents.reserve(asset.tangents.size() + totalVertexCount);
+        asset.texCoords0.reserve(asset.texCoords0.size() + totalVertexCount);
+        asset.texCoords1.reserve(asset.texCoords1.size() + totalVertexCount);
+        asset.indices.reserve(asset.indices.size() + totalIndexCount);
+
+        float2 minUV0 = float2(std::numeric_limits<float>::max());
+        float2 maxUV0 = float2(std::numeric_limits<float>::min());
```

Comment:
`numeric_limits::min` is a minimum positive value, you should use `numeric_limits::lower` here.